textbook/12/3/binomial.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>As the prefix bi implies  -&gt; As the prefix <em>bi-</em> implies</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>